CourseGrab/Home/HomeErrorView.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  HomeErrorView.swift
+//  CourseGrab
+//
+//  Created by Reade Plunkett on 3/4/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class HomeErrorView: UIView {
+
+    private let imageView = UIImageView()
+    private let subtitleLabel = UILabel()
+    private let titleLabel = UILabel()
+
+    init(frame: CGRect, title: String, subtitle: String, icon: UIImage) {
+        super.init(frame: frame)
+        backgroundColor = .white
```

Comment:
I would say ``// Setup appearance` to stay consistent with the rest of the codebase